cpp/oneapi/dal/algo/kmeans_init/backend/gpu/compute_kernel_distr_plus_plus_dense_dpc.cpp

```diff
@@ -339,7 +420,7 @@ template <typename Float>
 std::int64_t propose_sample_block(const sycl::queue& queue,
                                   std::int64_t feature_count,
                                   std::int64_t sample_count) {
-    return std::min<std::int64_t>(sample_count, 4096l);
+    return std::min<std::int64_t>(sample_count, 16'384l);
```

Comment:
16'384l ?

Reply:
And? Just better constant. Should be reviewed later.

Reply:
Sorry missed the question) yeah comments at least could be better or just readable variable